docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: in-progress
+- Start Date: 2015-08-10
+- RFC PR: #2047
+- Cockroach Issue:
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
+underlying key-value pairs that were accessed:
```

Comment:
It is surprising to me that `EXPLAIN DEBUG` will actually execute the query. Something like `EXPLAIN EXECUTE` or `EXPLAIN TRACE` would make this more clear (or something awkward like `EXPLAIN DEBUG_EXECUTE`, since I assume this is mainly for our own use and end users would prefer the cheaper plain `EXPLAIN`)